cssom-view-1/Overview.bs

```diff
@@ -1443,14 +1443,14 @@ The <dfn attribute for=Element>currentCSSZoom</dfn> attribute must return the
 
 <h3 id=element-scrolling-members>{{Element}} Scrolling Members</h3>
 
-To <dfn export>scroll a target into view</dfn> <var>target</var>, which is an <a for="/">Element</a> or <a>Range</a>,
+
+To <dfn export>determine the scroll-view position</dfn> of a <var>target</var>, which is an <a for="/">Element</a> or <a>Range</a>,
```

Comment:
nit: I think you meant scroll-into-view position?

Reply:
Yes, thanks. Done.